app/pb_kits/playbook/pb_background/background.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Playbook
+  module PbBackground
+    class Background
+      include Playbook::Props
+
+      partial "pb_background/background"
+
+      prop :background_color, type: Playbook::Props::Enum,
+                              values: %w[bg_gradient bg_dark bg_light white],
+                              default: "bg_light"
```

Comment:
@kre8sions @thestephenmarshall Should this be closer to the variable or be more simple. I am inclined to make them simple `gradient dark light white`.